README.md

```diff
@@ -498,17 +498,248 @@ group members to eventually converge on a single most preferred epoch.  We have
 not found a tie-breaking rule with all three properties, so this is future work.
 
 
-## 6. References
+## 6. Tangles
+
+A "tangle" in scuttlebutt is a way to define a directed acyclic graph (DAG) of 
+messages. You can read more about them in the [Tangle SIP].
+
+
+### 6.1. Members tangle
+
+The purpose of this tangle is to track the group membership (for a  particular 
+epoch of the group).
+
+We define the members tangle for some id `A` as:
+1. **candate messages**:
+   - the root message with id `A` so long as it has
+     ```javascript
+     content.tangles.members = { root: null, previous: null }
+     ```
+   - any messages where
+     ```javascript
+     content.tangles.members = { root: A, previous: [...] }
+     ```
```

Comment:
If I understood tangles correctly, the "candidates" are the input, and the "recipe" generates the output, which are the fields `content.tangles.members`. So it's not correct to put `content.tangles.members` as the candidates, because those are precisely the outcome of the recipe!

Reply:
I don't understand. The tangles data in `content.tangles.members` is the data needed for building the tangle (following the recipe). And if you want to add a new messages, then yes you bake a new message with something which will be consistent with the recipe outcome sofar?

Reply:
the recipe generates a tangle-graph. You can read the tips of this to generate the `previous` for any "next message"

Reply:
@staltz can we chat about this sync - text clarification of this feels too slow

Reply:
Yeah, the way you think of candidates is very confusing to me, and I bet I won't be the only one. Let's talk sync.